File: Angular2WebApplication/app/models/employee.ts
export class Department {
    details: string;
    name: string;
    id: number;
}

export class Position {
    details: string;
    name: string;
    id: number;
}


export class Employee {
    department: Department;
    position: Position;
    firstName: string;
    lastName: string;
    patronymic: string;
    photoUrl: string;
    dateOfBirth: Date;
    name: string;
    id: number;
}


export class GetListResponse<T> {
    items: T[];
    totalCount: number;
    pageNumber: number;
    pageSize: number;
}

File: Model/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    public class Employee : NamedObject
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Patronymic { get; set; }

        public virtual Position Position { get; set; }

        public virtual Department Department { get; set; }

        public string PhotoUrl { get; set; }

        public DateTime DateOfBirth { get; set; }

    }
}
